bangbangcon/bangbangcon.github.io
# Move 2018 site to its own directory; set up for 2019.

Set up the website for 2019:
- moved all the 2018 files to a new directory
- created a backdrop image with speakers from the past 5 years
- changed the background and links color
- updated pages with appropriate info

To test these changes:
- pull this branch
- make sure you have Jekyll
- run `jekyll s`
- go to `localhost:4000` and click through the page to make sure everything is in place
